### PR TITLE
Update profile for uefi machine

### PR DESCRIPTION
**Pull request by cedvid**
Update autoyast profile for 12-sp5 uefi.

- Related ticket: https://progress.opensuse.org/issues/154705

- Verification run:
    - mru-install-minimal-with-addons@uefi: https://openqa.suse.de/tests/13543021
    - mru-install-minimal-with-addons@aarch64-virtio: https://openqa.suse.de/tests/13545686

### CODE REVIEW

**Comment by rfan1** [comment]
Hello @cedvid, there is an existing autoyast xml file for x86_64 uefi already:)  

git-host:~/data/os-autoinst-distri-opensuse/data/autoyast_sle15/create_hdd # ll |grep uefi
-rw-r--r-- 1 root root 14333 Jan 22 23:25 create_hdd_gnome_x86_64_uefi.xml
-rw-r--r-- 1 root root 13954 Jan 22 23:25 create_hdd_textmode_x86_64_uefi.xml

**Comment by cedvid** [comment]
> -rw-r--r-- 1 root root 13954 Jan 22 23:25 create_hdd_textmode_x86_64_uefi.xml

@rfan1 Hi, I initially wanted to use the same profile for both uefi and non-uefi but you're right, I will edit the job group yaml with the existing one.

File: data/autoyast_qam/12-common_base_installation.xml.ep [hunk 1]
```diff
@@ -238,6 +238,9 @@
     <global>
       <timeout config:type="integer">-1</timeout>
     </global>
+    % if ($check_var->('UEFI', '1') and $check_var->('ARCH', 'x86_64')) {
+    <loader_type>grub2-efi</loader_type>
+    % }
   </bootloader>
```
Comment (rfan1) [inline data/autoyast_qam/12-common_base_installation.xml.ep:244]: In this case,  we may miss `aarch64`, can you please run VR on aarch64 to see if any regression issue?

File: data/autoyast_qam/12-common_base_installation.xml.ep [hunk 2]
```diff
@@ -301,8 +304,93 @@
   <partitioning config:type="list">
     <drive>
     <initialize config:type="boolean">true</initialize>
-    <use>all</use>
-    </drive>
```
Comment (rfan1) [inline data/autoyast_qam/12-common_base_installation.xml.ep:305]: The same comment as above one, can you please run VR for aarch64 platform as well?

File: data/autoyast_qam/12-common_base_installation.xml.ep (hunk 1 above)
Comment (cedvid) [reply data/autoyast_qam/12-common_base_installation.xml.ep:244]: Verification run on aarch64: https://openqa.suse.de/tests/13545686

**Review by dzedro — APPROVED** [review]

**Review by rfan1 — APPROVED** [review]

**Pull request merged**